common/wfRun.js

```diff
@@ -166,6 +167,22 @@ function hflowRun(opts, runCb) {
         }
     });
 
+    if (wfConfig.containerSpec) {
```

Comment:
I'm wondering where these `containerSpec` files should be published? Perhaps in workflow repositories, e.g. [montage2-workflow](https://github.com/hyperflow-wms/montage2-workflow)?